src/react/components/pages/editorPage/editorPage.tsx

```diff
@@ -311,8 +311,21 @@ export default class EditorPage extends React.Component<IEditorPageProps, IEdito
 
         this.setState({
             selectedAsset: assetMetadata,
-            assets: _.values(this.props.project.assets),
         });
+
+        if (!asset.parent) {
```

Comment:
As mentioned previously on other PR i don't believe we need this change since the child assets are already selected in the `onAssetMetadataChanged` event